ansible/deploy.yml

```diff
@@ -1,6 +1,14 @@
+- name: Config db server
+  hosts: db
+  become: yes
+  roles:
+    - db
+
 - name: Deploy Flask app and configure systemd service
   hosts: app
   become: yes
+  roles:
+    - app
   tasks:
```

Comment:
It's OK in this case.
But combining roles and tasks can be confusing.
Better use pre-tasks or post-tasks with roles.

Reply:
I thought it would be more compact, for the future I will divide it into pre-task and post-task, thank you